### DISCUSSION TITLE
Installer NewPipe sur Android 4.2.1

### DISCUSSION THREAD

**Discussion by Originale77** [TeamNewPipe/NewPipe#11674] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

Bonjour,
Je suis nouvelle sur votre site et un peu perdue.
J'ai une tablette Samsung qui refuse de se mettre à jour (Android 4.2.1). Je viens de découvrir NewPipe et je voulais l'installer sur ma tablette afin de visionner les vidéos de YouTube. Malheureusement, il semblerait que cela soit impossible pour des versions inférieures à Android 5+.
Auriez-vous une idée pour que je puisse encore utiliser ma tablette et ainsi contourner l'obsolescence programmée... ?
Merci d'avance pour toute réponse.
Cordialement

**Comment by aaferrari** [comment]
There is a [legacy](https://github.com/TeamNewPipe/NewPipe-legacy) version, but it hasn't been updated for a while and is currently archived. Doing some research I found a fork called [NewPipe-Legacy-Revo](https://github.com/ShareASmile/NewPipe-Legacy-Revo) maintained by another developer, although due to the versioning scheme it uses it is possible that it doesn't have all the features that have been added to the latest versions of NewPipe.

Another option could be [Skytube Legacy](https://github.com/SkyTubeTeam/SkyTubeLegacy) which runs on Android 4.0+ and seems to be on par with the main version.

**Reply by Originale77** [reply]
Merci beaucoup pour votre aide. Je vais consulter les liens que vous m'avez conseillés.
